lib/Subscribers/ImportExport/ImportExportFactory.php

```diff
@@ -100,20 +102,24 @@ public function formatFieldsForSelect2(
         [
           'id' => 'ignore',
           'name' => __('Ignore field...', 'mailpoet'),
+          'text' => __('Ignore field...', 'mailpoet'),
```

Comment:
Do we have to duplicate the fields on the PHP level? Could we only make the duplication on the javascript level and copy the values in there? So that we don't have to keep the duplication here and keep sending them?

Reply:
The duplication on the JavaScript level sounds good. I will try it.